tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -53,6 +53,12 @@ HEADSET_SSP_DAI_INDEX=2,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SP
 BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=8,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,USE_CHAIN_DMA=true,\
 DEEPBUFFER_FW_DMA_MS=10,DEEPBUFFER_D0I3_COMPATIBLE=true"
 
+"cavs-rt5682\;sof-mtl-rt1019-rt5682\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,\
+PDM1_MIC_B_ENABLE=1,DMIC0_PCM_ID=99,NUM_HDMIS=3,SPK_ID=6,PREPROCESS_PLUGINS=nhlt,\
```

Comment:
what happens if SPK_ID remains 7? why is this change necessary?

Reply:
mismatch the SPK_ID 7 due to non existed, because the mtl_rt1019_rt5682 backend dai id generated id from [0, 1, 2, 3, 4, 5, 6] mapping to [SSP2, dmic0, dmic1, HDMI1, HDMI2, HDMI3, SSP0]

according to 
...
		.name = "mtl_rt1019_rt5682",
		.driver_data = (kernel_ulong_t)(SOF_RT5682_MCLK_EN |
					SOF_RT5682_SSP_CODEC(2) |
					SOF_SPEAKER_AMP_PRESENT |
					SOF_RT1019_SPEAKER_AMP_PRESENT |
					SOF_RT5682_SSP_AMP(0) |
					SOF_RT5682_NUM_HDMIDEV(3)),